business-registry-dashboard/app/pages/[encodedOrgId]/affiliationInvitation/acceptToken/[token].vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
```

Comment:
I know, this file (and directory) looks weird. Here's why:

Nuxt handles routing in a very nice way. It discards the idea for router and instead, it does that automatically via directories and files (must be under the pages directory). Ref: https://nuxt.com/docs/getting-started/routing

The idea here, I want to grab: `'/:encodedOrgId/affiliationInvitation/acceptToken/:token'` where the first and last params are dynamic. Dynamic params are encapsulated by `[]`.

After doing that, if I want to access the token for example, I would do `const x = route.params.token`. Check line 30.

Reply:
What does a sample route look like?

Reply:
Something like: http://localhost:3000/en-CA/cmVzZA==/affiliationInvitation/acceptToken/eyJpZCI6NDI1LCJmcm9tT3JnSWQiOjY2OCwidG9PcmdJZCI6bnVsbCwiYnVzaW5lc3NJZGVudGlmaWVyIjoiQkMwODgzNjM5In0.ZyAKcw.fPYnDY8oJqbBjt-9EgvXxZtt3_s

Where:
encodedOrgId = cmVzZA==
token = eyJpZCI6NDI1LCJmcm9tT3JnSWQiOjY2OCwidG9PcmdJZCI6bnVsbCwiYnVzaW5lc3NJZGVudGlmaWVyIjoiQkMwODgzNjM5In0.ZyAKcw.fPYnDY8oJqbBjt-9EgvXxZtt3_s

Reply:
Well, that looks a lot like various unreadable shopping site URLs.

I'm a bit uneasy with using routes to pass this kind of data (but OK for basic identifiers). How about using URL parameters instead?

This search won't necessarily return best (or latest) practises but it might help. @thorwolpert can probably provide some guidance here, too.

https://www.google.com/search?q=how+to+form+url+routes ?

Reply:
So for this Sev, you were thinking of getting rid of this whole directory and file and instead handle everything in the index.vue file by reading the URL parameters?

Thor, what do you think? @thorwolpert

Reply:
Yes, that's what I was thinking.

Curious, though. Would Nuxt have created "virtual directories" for each token?